docs/troubleshooting.md

```diff
@@ -83,14 +83,14 @@ Check the logs for errors
 kubectl logs daemonset/wavefront-logging -n observability-system
 ```
 
-## Missing or incomplete data flowing into Tanzu Observability
+## Missing or incomplete data flowing into Operations for Applications
 
 If you are experiencing gaps in data, where expected metrics or metric tags are not showing, please review the following steps.
 
 **Note:** For out of the box Kubernetes Control Plane dashboard, certain managed Kubernetes environments do not support collecting metrics of all control plane elements. For detailed information, please refer to our [supported metrics page](https://github.com/wavefrontHQ/observability-for-kubernetes/blob/main/docs/metrics.md#control-plane-metrics).
 
 ### Are all components Healthy?
-Check wavefront integration status locally to determine if all components are healthy by using the below command,
+Check Observability for Kubernetes Operator status locally to determine if all components are healthy by using the below command,
```

Comment:
Check the Observability for Kubernetes Operator status locally to determine if all components are healthy by using the below command: